Client/game_sa/CFireSA.cpp

```diff
@@ -242,13 +242,16 @@ static void _declspec(naked) HOOK_CFire_Extinguish()
     _asm
     {
         mov [eax+730h], edi
+
+        push ebx
         mov ebx, [esp+8]
```

Comment:
Since you modify `esp` with the `push ebx` above now, don't you have to fix the `+8` offset here?

Reply:
Yeah, you're right i forgot about it.